bears/vcs/git/GitCommitBear.py

```diff
@@ -60,9 +60,7 @@ def run(self,
             self.err("git:", repr(stderr))
             return
 
-        # git automatically removes trailing whitespaces. Also we need to
-        # remove the last \n printed to align the prompt onto the next line.
-        stdout = stdout.splitlines()[:-1]
+        stdout = stdout.rstrip("\n").splitlines()
```

Comment:
please do a strip, possibly there's a leading newline due to some reason as we're not getting the commit message directly but via pipes